### ISSUE TITLE
bamboo chest rendering

### ISSUE THREAD

**Issue by cbaakman** [Tropicraft/Tropicraft#103]
The double bamboo chest sometimes turns invisible. It happens when the first placed block is in the screen, but the second placed block isn't.

**Comment by cbaakman** [comment]
It seems that it helps to extend the render bounding box for large chests:

```
public class TileEntityBambooChest extends TileEntityChest {
..
    @Override
    public AxisAlignedBB getRenderBoundingBox ()
    {
    	checkForAdjacentChests();

    	if (this.adjacentChestZPos != null)
    	{
    		return AxisAlignedBB.getBoundingBox(xCoord, yCoord, zCoord, xCoord + 1, yCoord + 1, zCoord + 2);
    	}
    	else if (this.adjacentChestXPos != null)
        {
    		return AxisAlignedBB.getBoundingBox(xCoord, yCoord, zCoord, xCoord + 2, yCoord + 1, zCoord + 1);
        }
    	else
    	{
    		return AxisAlignedBB.getBoundingBox(xCoord, yCoord, zCoord, xCoord + 1, yCoord + 1, zCoord + 1);
    	}
    }
}
```

**Comment by Cojomax99** [comment]
Fixed in #104
